src/admin-views/edit_admin_panel.php

```diff
@@ -59,50 +60,53 @@
 					<div class="input_block">
-						<label class="ticket_form_label" for="ticket_description"><?php esc_html_e( 'Ticket Description:', 'event-tickets' ); ?></label>
-						<textarea rows="5" cols="40" name="ticket_description" class="ticket_field" id="ticket_description"></textarea>
+						<label class="ticket_form_label ticket_form_left" for="ticket_description" ><?php esc_html_e( 'Ticket Description:', 'event-tickets' ); ?></label>
+						<textarea rows="5" cols="40" name="ticket_description" class="ticket_field ticket_form_right" id="ticket_description"></textarea>
 						<div class="input_block">
-							<label><input type="checkbox" name="tribe_show_description" value="1"> Show description on front end and emailed tickets.</label>
+							<label class="tribe_soft_note"><input type="checkbox" name="tribe_show_description" value="1" class="ticket_form_left"> Show description on front end and emailed tickets.</label>
 						</div>
 					</div>
 					<div class="input_block">
-						<label class="ticket_form_label" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
-						<input autocomplete="off" type="text" class="ticket_field" size='10' name="ticket_start_date" id="ticket_start_date" value="" >
-						<span class="ticket_start_time ticket_time">
-							<?php echo tribe_get_datetime_separator(); ?>
-							<select name="ticket_start_hour" id="ticket_start_hour" class="ticket_field tribe-dropdown">
-								<?php echo $startHourOptions; ?>
-							</select>
-							<select name="ticket_start_minute" id="ticket_start_minute" class="ticket_field tribe-dropdown">
-								<?php echo $startMinuteOptions; ?>
-							</select>
-							<?php if ( ! strstr( get_option( 'time_format', Tribe__Date_Utils::TIMEFORMAT ), 'H' ) ) : ?>
-								<select name="ticket_start_meridian" id="ticket_start_meridian" class="ticket_field tribe-dropdown">
-									<?php echo $startMeridianOptions; ?>
+						<label class="ticket_form_label ticket_form_left" for="ticket_start_date"><?php esc_html_e( 'Start sale:', 'event-tickets' ); ?></label>
+						<div class="ticket_form_right">
+							<input autocomplete="off" type="text" class="ticket_field" size='10' name="ticket_start_date" id="ticket_start_date" value="" >
+							<span class="ticket_start_time ticket_time">
+								<?php echo tribe_get_datetime_separator(); ?>
+								<select name="ticket_start_hour" id="ticket_start_hour" class="ticket_field tribe-dropdown">
+									<?php echo $startHourOptions; ?>
 								</select>
-							<?php endif; ?>
-						</span>
+								<select name="ticket_start_minute" id="ticket_start_minute" class="ticket_field tribe-dropdown">
+									<?php echo $startMinuteOptions; ?>
+								</select>
+								<?php if ( ! strstr( get_option( 'time_format', Tribe__Date_Utils::TIMEFORMAT ), 'H' ) ) : ?>
+									<select name="ticket_start_meridian" id="ticket_start_meridian" class="ticket_field tribe-dropdown">
```

Comment:
Yeah, let's do that in a separate ticket/PR so we can be sure we're not changing something that isn't in the database and doesn't break compatibility.